docs/blog/articles/2024-05-15-Kserve-0.13-release.md

```diff
@@ -0,0 +1,123 @@
+
+Below are a summary of the key changes.
+
+## Enhanced HuggingFace Runtime Support
+
+![kserve-components](../../images/kserve_new.png)
+
+KServe v0.13 enriches its Huggingface runtime and now supports running HuggingFace models out-of-the-box. KServe v0.13 implements a [KServe HuggingFace Serving Runtime](https://github.com/kserve/kserve/tree/master/python/huggingfaceserver), `kserve-huggingfaceserver`. With this implementation, KServe can now automatically infer a [task](https://huggingface.co/tasks) from model architecture and select the optimized serving runtime. Currently supported tasks include sequence classification, token classification, fill mask, text generation, and text to text generation.
+
+![kserve-huggingface](../../images/kserve-huggingface.png)
+
+Here is an example to serve BERT model by deploying an Inference Service with HuggingFace runtime for classification task.
+
+```yaml
+apiVersion: serving.kserve.io/v1beta1
+kind: InferenceService
+metadata:
+  name: huggingface-bert
+spec:
+  predictor:
+    model:
+      modelFormat:
+        name: huggingface
+      args:
+      - --model_name=bert
+      - --model_id=bert-base-uncased
+      - --tensor_input_names=input_ids
+      resources:
+        limits:
+          cpu: "1"
+          memory: 2Gi
+          nvidia.com/gpu: "1"
+        requests:
+          cpu: 100m
+          memory: 2Gi
+          nvidia.com/gpu: "1"
+```
+
+You can also deploy BERT on the more optimized inference runtime like Triton using Hugging Face Runtime for pre/post processing, see more details [here](https://kserve.github.io/website/master/modelserving/v1beta1/triton/huggingface/).
+
+
+### vLLM support 
+
+Version 0.13 introduces dedicated runtime support for [vLLM](https://docs.vllm.ai/en/latest/), for enhanced transformer model serving. This support now includes auto-mapping vLLMs as the backend for supported tasks, streamlining the deployment process and optimizing performance. If vLLM is not supported by a task it will default to the HuggingFace backend. See example below. 
+
+```yaml
+apiVersion: serving.kserve.io/v1beta1
+kind: InferenceService
+metadata:
+  name: huggingface-llama2
+spec:
+  predictor:
+    model:
+      modelFormat:
+        name: huggingface
+      args:
+      - --model_name=llama2
+      - --model_id=meta-llama/Llama-2-7b-chat-hf
+      resources:
+        limits:
+          cpu: "6"
+          memory: 24Gi
+          nvidia.com/gpu: "1"
+        requests:
+          cpu: "6"
+          memory: 24Gi
+          nvidia.com/gpu: "1"
+```
+
+See more details in our updated docs to [Deploy the Llama2 model with Hugging Face LLM Serving Runtime](https://kserve.github.io/website/master/modelserving/v1beta1/llm/huggingface/).
+
+Additionally, if the HuggingFace backend is preferred over vLLM, vLLM auto-mapping can be disabled with the `--backend=huggingface` arg.
```

Comment:
@yuzisun is this right or should it be --disable-vllm ?

Reply:
This is correct. We have moved away from `--disable-vllm` to use `--backend`